packages/asset-swapper/src/utils/market_operation_utils/types.ts

```diff
@@ -104,6 +105,10 @@ export interface BalancerFillData extends FillData {
     poolAddress: string;
 }
 
+export interface CreamFillData extends FillData {
```

Comment:
Should we  just extend `BalancerFillData`?

Reply:
got rid of CreamFillData altogether